packages/plugin-spelling/src/rules/cspell.ts

```diff
@@ -1,6 +1,6 @@
 import { textLanguage } from "@flint.fyi/text";
 import { parseJsonSafe } from "@flint.fyi/utils";
-import { DocumentValidator } from "cspell-lib";
+import { type DocumentValidator } from "cspell-lib";
```

Comment:
I know we don't use verbatimModuleSyntax, but it's a) cleaner this way and b) it's officially ts team recommendation to use it so we'll probably do that someday.

```suggestion
import type { DocumentValidator } from "cspell-lib";
```

Reply:
I just realized this was on my own change. Silly me 😅

Anyway, point is perhaps worth considering [tseslint/no-import-type-side-effects](https://typescript-eslint.io/rules/no-import-type-side-effects)

Reply:
Haha. Yeah. I was thinking of filing an issue to enable it. And also to switch to `.ts` extensions. WDYT?